components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt

```diff
@@ -121,28 +116,28 @@ object Nimbus : NimbusApi {
 
         this.onExperimentUpdated = onExperimentUpdated
 
+        // Build Nimbus AppContext object to pass into initialize
+        val experimentContext = buildExperimentContext(context)
+
+        // Build a File object to represent the data directory for Nimbus data
+        dataDir = File(context.applicationInfo.dataDir, NIMBUS_DATA_DIR)
+
+        // Initialize Nimbus
+        nimbus = NimbusClient(
+            experimentContext,
+            dataDir.path,
+            RemoteSettingsConfig(
+                serverUrl = context.resources.getString(R.string.nimbus_default_endpoint),
+                bucketName = EXPERIMENT_BUCKET_NAME,
+                collectionName = EXPERIMENT_COLLECTION_NAME
+            ),
+            // The "dummy" field here is required for obscure reasons when generating code on desktop,
+            // so we just automatically set it to a dummy value.
+            AvailableRandomizationUnits(clientId = null, dummy = 0)
+        )
+
```

Comment:
Separating construction of the `NimbusClient` from the fetching of the library is necessary to allow Nimbus to be included in the component tree.